docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -0,0 +1,28 @@
+# Kubeadm control plane
+
+Using the Kubeadm control plane type to manage a control plane provides several ways to upgrade control plane machines.
+
+## Upgrading
+
+The high level steps to fully upgrading a workload cluster are to first upgrade the control plane and then upgrade
+the worker machines.
+
+### Upgrading workload machines managed by a `MachineDeployment`
+
+It is recommended to manage workload machines with a `MachineDeployment` as this will transparently manage `MachineSet`s
+and `Machine`s to allow for a seamless scaling experience. A modification to the `MachineDeployment`s spec will begin a
```

Comment:
I can't come up with better wording at the moment, but I think it might be better to give a better description of how the MachineDeployment is orchestrating the rollout of the change (and allowing for the ability to rollback in the event of an issue)